src/MakersOfDenmark.Domain/Models/User/MODUser.cs

```diff
@@ -0,0 +1,40 @@
+﻿using MakersOfDenmark.Domain.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MakersOfDenmark.Domain.Models.User
+{
+    public class MODUser : Entity<Guid>
```

Comment:
Er MODUser bedre end bare 'User'? Vi kan dog sagtens bruge MODUser hvis du/I foretrækker dette. 

Ved ikke om mappe/namespace (MakersOfDenmark.Domain.Models.User) skal gå ned i ...Models.MODUser i stedet for eller om det er ligegyldigt.

Reply:
Jeg synes vi blot skal kalde den for User.

Reply:
MOD = MakersOfDenmark

Hvis vi på et tidspunkt skal bruge Identity, så bliver det noget rod hvis vi har en entity der hedder User :)

Reply:
Det er sandt!